### PR TITLE
Introduce licensing.BatchChanges (alias Campaigns to it)

### PR DESCRIPTION
**Pull request by mrnugget**
This introduces licensing.BatchChanges as a new licensing feature tag.

Since we're only deprecating the old tag, we'll fall back to check licensing.Campaigns.

### CODE REVIEW

**Review by eseliger — APPROVED** [review]
I don't know how this works, are we issuing the new licenses with the new tag now? Or is this a manual process?

File: enterprise/internal/licensing/data.go [hunk 1]
```diff
@@ -60,6 +64,7 @@ var planFeatures = map[Plan][]Feature{
 		FeatureRemoteExtensionsAllowDisallow,
 		FeatureBranding,
 		FeatureCampaigns,
+		FeatureBatchChanges,
 		FeatureMonitoring,
 		FeatureBackupAndRestore,
```
Comment (mrnugget) [inline enterprise/internal/licensing/data.go:69]: @malomarrec Just want to highlight this here (expand the diff to see context better): I've also included `BatchChanges` in the `oldEnterprise` plan. I'm not up to date on pricing/tiers discussions, but this means that "batch-changes" would act like the old "campaigns" tag did.

**Comment by mrnugget** [comment]
> I don't know how this works, are we issuing the new licenses with the new tag now? Or is this a manual process?

I'm reasonably sure it's a manual process.

File: enterprise/internal/licensing/data.go [hunk 2]
```diff
@@ -40,9 +40,13 @@ const (
 	// FeatureBranding is whether custom branding of this Sourcegraph instance has been purchased.
 	FeatureBranding Feature = "branding"
 
-	// FeatureCampaigns is whether campaigns on this Sourcegraph instance has been purchased.
+	// DEPRECATED: See FeatureBatchChanges.
+	// FeatureCampaigns is whether campaigns (now: batch changes) on this Sourcegraph instance has been purchased.
```
Comment (unknwon) [inline enterprise/internal/licensing/data.go:44]:
```suggestion
	// FeatureCampaigns is whether campaigns (now: batch changes) on this Sourcegraph instance has been purchased.
	//
	// DEPRECATED: See FeatureBatchChanges.
```

If you order lines this way, some IDE (e.g. GoLand) would visually strikethrough the word to identify its deprecation.

**Review by unknwon — APPROVED** [review]

File: enterprise/internal/licensing/data.go (hunk 2 above)
Comment (mrnugget) [reply enterprise/internal/licensing/data.go]: Cool! I was wondering about that.

**Pull request merged**